common/config.ts

```diff
@@ -78,6 +78,37 @@ export const UNIT_VISIBILITIES: ValueName[] = [
     { value: 'fremen', name: "Fremenská" }
 ]
 
+export const ALL_DECREE: string[] = [
+    "Mandát sardaukarů",
+    "Zrušení mandátu sardaukarů",
+    "Vyslání vojsk Landsraadu",
+    "Změna mandátu vojsk Landsraadu",
+    "Stažení vojsk Landsraadu",
+    "Zřízení vyšetřovací komise",
+    "Přerozdělení podílů CHOAM",
+    "Uvalení embarga",
+    "Zrušení embarga",
+    "Změna kvót",
+    "Jiný",
+    "- Žádný -"
+]
+
+export const SPECIAL_DECREE: string[] = [
+    "Bill of Particulars",
+    "- Žádný -"
+]
+
+export const NEUTRAL_HOUSES: string[] = [
```

Comment:
This info is already in db - voting rights. I wouldn't duplicate it here.

Reply:
In that case we have to add property for each voting right if it is neutral house. But I can remove it from this PR then because that is for the part 3 of "decret changes".

Reply:
Yes property is better than this hardcoding. But let's do it in another PR.

Reply:
Removed.